site/content/documentation/reference/_index.md

```diff
@@ -3,5 +3,5 @@ title: "Reference"
 weight: 4
 ---
 
+- [RDF4J REST API](/documentation/reference/rest-api/)
 - [RDF4J API Javadoc](/javadoc/latest)
-- older releases: [3.7.7](/javadoc/3.7.7), [3.6.3](/javadoc/3.6.3), [3.5.0](/javadoc/3.5.0), [3.4.3](/javadoc/3.4.3), [3.3.0](/javadoc/3.3.0), [3.2.0](/javadoc/3.2.0), [3.1.0](/javadoc/3.1.0), [3.0.4](/javadoc/3.0.4), [2.5.4](/javadoc/2.5.4)
```

Comment:
3.7.7 is probably still relevant since it's the newest version for those stuck on Java 8.

Reply:
Ah yeah it's not that I want to remove the actual javadocs, per se. But this long list of links is not really necessary IMO (plus, we always forget to update it anyway - not how any releases post 3.7.7 are missing).